wiki/misc/mappings/en.md

```diff
@@ -0,0 +1,99 @@
+# Customizing the Mappings in Your Development Environment
+
+## What are mappings?
+
+Before a new Minecraft version jar is published to Mojang's servers, it goes through a process called *obfuscation*,
+where the human-readable class, field and method names are simplified to just a few letters, mainly to optimize the file
+size. In addition, obfuscation makes code very difficult to understand, because most names are replaced by a few random
```

Comment:
```suggestion
size. In addition, obfuscation makes code very difficult to understand, because those simplified names aren't just contractions of the real names, they're completely random
```
made the wording a little more fun

Reply:
Whoops